Test/algo_test.py

```diff
@@ -29,18 +29,18 @@ def test_text_unicode(self):
         self.assertEquals('text', result2.metadata.content_type)
         self.assertEquals(telephone, result2.result)
 
-    def test_json_unicode(self):
-        telephone = [u"\u260E"]
-
-        #Unicode input to pipe()
-        result1 = self.client.algo('util/Echo').pipe(telephone)
-        self.assertEquals('json', result1.metadata.content_type)
-        self.assertEquals(telephone, result1.result)
-
-        #Unicode return in .result
-        result2 = self.client.algo('util/Echo').pipe(result1.result)
-        self.assertEquals('json', result2.metadata.content_type)
-        self.assertEquals(telephone, result2.result)
+    # def test_json_unicode(self):
```

Comment:
@pmcq @jamesatha if you guys have some insights into this error case that would be great - it doesn't seem to succeed at all anymore and I'm unsure as to what changed.

Reply:
i tried running these locally and it seems like the client isn't sending the initial request as JSON any longer but as a text string, i'm not sure if these tests just haven't been run for awhile or what though.

Reply:
yea that's what I was thinking - I don't see anything specific that it's testing but I also don't want to remove a critical test that we've just forgotten about over the years.